templates/zerver/help/changing-the-default-language.md

```diff
@@ -0,0 +1,27 @@
+# Changing the Default Language
+
+Zulip is written primarily in English, but includes feature support for a variety of languages. Changing the default language in Zulip will translate many of the settings and menu options into the language that you select. Because some languages are only partially supported,  you may still see English words interspersed throughout Zulip.
```

Comment:
either omit the comma here or say ", but it includes".  Commas separate two independent clauses.  I know this is very pedantic.